### PR TITLE
Added privacy to footer

### PR DESCRIPTION
**Pull request by morsingnet**
Added Privacy footer link
Fixed spelling
updated CTA on community plan

### CODE REVIEW

File: src/components/PlanCardHead.vue [hunk 1]
```diff
@@ -9,7 +9,7 @@
       <p class="card-text">{{ plan.description }}</p>
     </div>
     <div class="card-footer">
-      <SelectPlanButton :plan="plan" text="Start Application" />
+      <SelectPlanButton :plan="plan" text="Enroll" />
```
Comment (bsommardahl) [inline src/components/PlanCardHead.vue:12]: We had agreed that "apply" was better than "enroll". This reverts that change.

File: src/views/Community.vue [hunk 2]
```diff
@@ -24,7 +24,7 @@
 
     <div class="row">
       <div class="col text-center">
-        <SelectPlanButton :plan="getCommunityPlan" text="Start Application" />
+        <SelectPlanButton :plan="getCommunityPlan" text="Enroll" />
       </div>
```
Comment (bsommardahl) [inline src/views/Community.vue:28]: We had agreed that "apply" was better than "enroll". This reverts that change.

File: src/views/Community.vue [hunk 3]
```diff
@@ -151,7 +151,7 @@
...
```
Comment (bsommardahl) [inline src/views/Community.vue:155]: We had agreed that "apply" was better than "enroll". This reverts that change.

File: src/views/landing/StartApplicationForm.vue [hunk 4]
```diff
@@ -58,11 +58,17 @@
...
-      <button class="btn btn-lg btn-cta btn-block text-uppercase" type="submit">Start Application</button>
-    </div>
-    <div class="privacy text-center mt-3">
-      <router-link to="/policy/privacy">Privacy Policy</router-link>
+          <div class="cta">
+            <button
+              class="btn btn-lg btn-cta btn-block text-uppercase"
+              type="submit"
+            >Enroll</button>
+          </div>
```
Comment (bsommardahl) [inline src/views/landing/StartApplicationForm.vue:66]: We had agreed that "apply" was better than "enroll". This reverts that change.

Comment (morsingnet) [reply src/views/landing/StartApplicationForm.vue:66]: Ok - I was changing it from "Start Application" to match homepage (back then Enroll).
Ill do another PR.

**Comment by morsingnet** [comment]
Creating a new PR with new CTAs and spelling fixes

**Pull request closed without merging**